README.md

```diff
@@ -97,147 +97,136 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 ### FancyZones
 
- - Set the process and main thread priority to normal.
- - Fixed handling newly created windows on Windows 11.
- - Fixed scenarios where opening the FancyZones Editor would reset the layouts.
+ - Fixed an issue causing context menu pop-ups from some apps to automatically snap to a zone. (This was a hotfix for 0.74)
+ - Applied the fix for the context menu pop-ups to the logic that decides which windows can be snapped.
+ - Reworked the "Keep windows in their zones" option to include the work area and turn it on by default, fixing an incompatibility with the Copilot toolbar.
+ - Fixed an issue causing windows to be snapped while moving to a different virtual desktop.
 
 ### File Explorer add-ons
 
- - Optimized CPU usage for generating SVG thumbnails.
- - Improved handling of Gcode Thumbnails, including JPG and QOI formats. Thanks [@pedrolamas](https://github.com/pedrolamas)!
- - Better handled errors when sending telemetry, which were causing reported crashes.
- - Fixed some thumbnails not being shown centered like before the optimization.
-
-### File Locksmith
+ - Fixed an issue blocking some SVG files from being previewed correctly. (This was a hotfix for 0.74)
+ - Fixed crashes on invalid files in the STL Thumbnail generator.
 
- - Shows files opened by processes with PID greater than 65535. Thanks [@poke30744](https://github.com/poke30744)!
- - Fixed a GDI object leak in the context menu which would crash Explorer.
- 
-### Find My Mouse
+### GPO
 
- - Added new activation methods, including by hotkey. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Added a global GPO rule that applies for all utilities unless it's overridden. Thanks [@htcfreek](https://github.com/htcfreek)!
+ - Added GPO rules to control which plugins should be enabled/disabled by policy. Thanks [@htcfreek](https://github.com/htcfreek)!
 
-### Hosts File Editor
+### Image Resizer
 
- - Ignore the default ACME sample entries in the hosts file. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Improved save error handling and added better error messages. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Corrected a check for an error when signaling the application to start as administrator.
- - Refactored the context menu. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
- - Fixed dialogs overlapping the title bar after the upgrade to Windows App SDK 1.4. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Fixed wrong .bmp file association in the registry. Thanks [@meitinger](https://github.com/meitinger)!
 
 ### Keyboard Manager
 
- - Distinguish between the regular minus key and the numpad minus key.
+ - Visually distinguish between the Numpad and regular dot characters in the UI.
+ - This utility is now disable by default on new installs, since it requires user configuration to do anything useful.
```

Comment:
```suggestion
 - This utility is now disabled by default on new installations, since it requires user configuration to affect keyboard behavior.
```